plugins/cost-insights/src/components/CostOverviewCard/CostOverviewByProductChart.tsx

```diff
@@ -144,18 +175,24 @@ export const CostOverviewByProductChart = ({
   }) => {
     if (isInvalid({ label, payload })) return null;
 
-    const title = dayjs(label).utc().format(DEFAULT_DATE_FORMAT);
+    const dateTitle = dayjs(label).utc().format(DEFAULT_DATE_FORMAT);
     const items = payload.map(p => ({
       label: p.dataKey as string,
       value: formatGraphValue(p.value as number),
       fill: p.fill!,
     }));
-
+    const expandText = (
+      <Box display="flex" justifyContent="space-between" alignItems="center">
+        <FullScreenIcon />
+        <Typography>Click to expand</Typography>
+      </Box>
+    );
     return (
-      <Tooltip title={title}>
+      <Tooltip title={dateTitle}>
         {items.reverse().map((item, index) => (
           <TooltipItem key={`${item.label}-${index}`} item={item} />
         ))}
+        {!isExpanded ? expandText : null}
```

Comment:
Minor consistency tweak; should have a divider?

![breakdown-tooltip](https://user-images.githubusercontent.com/556258/105442585-250cae00-5c27-11eb-8e68-a5f1f4eb8c42.png)